### PR TITLE
Docs: en translation

### PR DESCRIPTION
**Pull request by Shimogawa**

### CODE REVIEW

File: docs/pages/guide/post.md [hunk 1]
```diff
@@ -5,13 +5,21 @@ categories:
   - Guide
 ---
 
-## FrontMatter
+## Front Matter
```
Comment (YunYouJun) [inline docs/pages/guide/post.md]: I think we can use `FrontMatter` directly for searching.

File: docs/pages/guide/page.md [hunk 2]
```diff
@@ -5,9 +5,15 @@ categories:
   - Guide
 ---
 
-## Frontmatter
```
Comment (YunYouJun) [inline docs/pages/guide/page.md:8]: I think we can use FrontMatter directly for searching.

**Review by YunYouJun — CHANGES_REQUESTED** [review]

**Comment by YunYouJun** [comment]
That's amazing. Thanks for your work.

**Comment by flaribbit** [comment]
好快

**Review by YunYouJun — APPROVED** [review]

**Comment by YunYouJun** [comment]
Just a reference https://jekyllrb.com/docs/front-matter/ .

**Pull request merged**